quax/examples/zero/_core.py

```diff
@@ -171,7 +171,7 @@ def _(lhs: Zero, rhs: Zero, **kwargs) -> Zero:
 def _integer_pow(x: Zero, *, y: int) -> Union[Array, Zero]:
     # Zero is a special case, because 0^0 = 1.
     if y == 0:
-        return jnp.ones(x.shape, x.dtype)
+        return jnp.ones(x.shape, x.dtype)  # pyright: ignore
```

Comment:
What's going on with this one?
Other than this, this LGTM!

Reply:
Pyright was complaining that the stripe argument is the wrong type.

Reply:
Hmm, that sounds like a JAX bug. Anyway, no big deal, LGTM!